### DISCUSSION TITLE
[Feature] Time period to auto upload to shared album

### DISCUSSION THREAD

**Discussion by hkfuertes** [immich-app/immich#12346] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

(First of all, sorry if it was indeed repeated, as far as I have search, I have not seen any discussion as this one)

My feature proposal is mainly one, but with an extra (would be nice). Recently I went on a trip with friends, and we all had immich app installed and running against my server, so any photo taken with any of our smartphones was automatically synced, but we had to manually add the uploaded fotos to the shared album.

Could there be a way to specify auto share to an album in a time window (say, the duration of the trip)?

And the extra... Weddings... can this also be done somehow with guests/temporary users?

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1673 (see also #11668)

**Reply by hkfuertes** [reply]
Oh thank you!
Sorry for the inconvenience!
